docs/modules/release-notes/pages/all-releases/4_13_3S.adoc

```diff
@@ -0,0 +1,107 @@
+= Kobiton 4.13.3S release notes
+:navtitle: Kobiton 4.13.3S release notes
+
+_July 19, 2024_
+
+== New OS support
+
+kobiton 4.13.3S add support for the following:
+
+* iOS/iPadOS 18 Beta
+* Xcode 16 Beta
+
+== CoreDevice: Lightning mode support for iOS 17.4 devices and later without networking changes
+
+When Apple introduced some major changes with the rollout of iOS 17, it requires significant network configuration to support Lightning mode with Kobiton. The additional network requirements are hard to implement and sometimes pose a security risks for some organizations.
+
+With this release, we are happy to introduce CoreDevice, which is a solution to the above problem. With CoreDevice, iOS 17.4 and later devices can have Lightning mode without the additional network requirements.

```

Comment:
Replace this sentence with: 
With iOS 17.4, Apple introduced the CoreDevice communication protocol that significantly enhances connections between iOS devices and the Kobiton platform. We implemented this innovative protocol with this release. CoreDevice eliminates the need for the special Lightning Mode networking changes introduced with iOS 17 for iOS 17.4+ devices, simplifying your networking.